scipy/linalg/tests/test_lapack.py

```diff
@@ -138,6 +138,66 @@ def test_clapack(self):
 
 class TestLeastSquaresSolvers(TestCase):
 
+    def test_gels(self):
+        for dtype in REAL_DTYPES:
+            a1 = np.array([[1.0,2.0],
+                          [4.0,5.0],
+                          [7.0,8.0]], dtype=dtype)
+            b1 = np.array([16.0, 17.0, 20.0], dtype=dtype)
+            gels, gels_lwork = get_lapack_funcs(('gels','gels_lwork'),
+                                                (a1, b1))
+
+            m, n = a1.shape
+            if len(b1.shape) == 2:
+                nrhs = b1.shape[1]
+            else:
+                nrhs = 1
+
+            # Request of sizes
+            work, info = gels_lwork(m,n,nrhs)
+            lwork = int(np.real(work))
+
+            lqr, x, info = gels(a1, b1, lwork)
+            assert_allclose(x[:-1], np.array([-14.333333333333323,
+                                              14.999999999999991], dtype=dtype),
+                            rtol=25*np.finfo(dtype).eps)
+            assert_allclose(lqr, np.array([[-8.1240384 , -9.6011363 ],
+                                           [ 0.43840236,  0.90453403],
+                                           [ 0.76720414,  0.90907633]],
+                                          dtype=dtype),
+                            rtol=25*np.finfo(dtype).eps)
```

Comment:
https://travis-ci.org/scipy/scipy/jobs/227275929 looks real. On my machine the abs difference is of the order of 1e-9.